tests/utils/throttleTick.test.js

```diff
@@ -0,0 +1,77 @@
+/* global assert, suite, test */
+var throttleTick = require('utils').throttleTick;
+
+suite('utils.throttleTick', function () {
+  var ts;
```

Comment:
I tried this, but function spies don't seem to do the same as method spies (i.e. when you call the function, the spy doesn't get called) -- so I'm giving up on this part for now, feel free to take it up.
``` Spy objects are objects returned from `sinon.spy()`. When spying on existing methods with `sinon.spy(object, method)`, the following properties and methods are also available on `object.method`. ```